website/components/ApiMeta.tsx

```diff
@@ -1,4 +1,4 @@
-import { useLang } from 'rspress/runtime';
+import semver from 'semver';
```

Comment:
Should we use `semver` in the client code? Can you measure the difference in bundle size?